slotfills/sections/richtext-field.tsx

```diff
@@ -1,17 +1,30 @@
 /* global tinyMCEPreInit */
 import React, { useState } from 'react';
-import PropTypes from 'prop-types';
 import { PanelRow } from '@wordpress/components';
 import { Editor } from '@tinymce/tinymce-react';
+import Field from '@/interfaces/field';
 
-const RichtextField = ({
+declare global {
+  const tinyMCEPreInit: {
+    baseURL: string,
+  };
+}
+
+type Props = {
+  field: Field,
+  valueHook: Function,
+  index: Number,
```

Comment:
Should be able to do lowercase `number` here - and prefer primitives when possible.